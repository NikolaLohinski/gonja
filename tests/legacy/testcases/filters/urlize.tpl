{{ "http://www.john-doe.de"|urlize|safe }}
{{ "http://www.john-doe.de"|urlize(rel='nofollow')|safe }}
{{ "http://www.john-doe.de"|urlize(rel='nofollow', target='_blank')|safe }}
{{ "http://www.john-doe.de"|urlize(rel='noopener')|safe }}
{{ "www.john-doe.de"|urlize|safe }}
{{ "john-doe.de"|urlize|safe }}
--
{% filter urlize|safe %}
Please mail me at demo@example.com or visit mit on:
- lorem ipsum github.com/nikolalohinski/gonja lorem ipsum
- lorem ipsum http://www.john-doe.de lorem ipsum
- lorem ipsum https://www.john-doe.de lorem ipsum
- lorem ipsum https://www.john-doe.de lorem ipsum
- lorem ipsum www.john-doe.de lorem ipsum
- lorem ipsum www.john-doe.de/test="test" lorem ipsum
{% endfilter %}
--
{% filter urlize(target='_blank', rel="nofollow")|safe %}
Please mail me at demo@example.com or visit mit on:
- lorem ipsum github.com/nikolalohinski/gonja lorem ipsum
- lorem ipsum http://www.john-doe.de lorem ipsum
- lorem ipsum https://www.john-doe.de lorem ipsum
- lorem ipsum https://www.john-doe.de lorem ipsum
- lorem ipsum www.john-doe.de lorem ipsum
- lorem ipsum www.john-doe.de/test="test" lorem ipsum
{% endfilter %}
--
{% filter urlize(15)|safe %}
Please mail me at demo@example.com or visit mit on:
- lorem ipsum github.com/nikolalohinski/gonja lorem ipsum
- lorem ipsum http://www.john-doe.de lorem ipsum
- lorem ipsum https://www.john-doe.de lorem ipsum
- lorem ipsum https://www.john-doe.de lorem ipsum
- lorem ipsum www.john-doe.de lorem ipsum
- lorem ipsum www.john-doe.de/test="test" lorem ipsum
{% endfilter %}
