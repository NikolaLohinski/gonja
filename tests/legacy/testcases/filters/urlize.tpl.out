<a href="http://www.john-doe.de" rel="noopener">http://www.john-doe.de</a>
<a href="http://www.john-doe.de" rel="nofollow noopener">http://www.john-doe.de</a>
<a href="http://www.john-doe.de" rel="nofollow noopener" target="_blank">http://www.john-doe.de</a>
<a href="http://www.john-doe.de" rel="noopener">http://www.john-doe.de</a>
<a href="http://www.john-doe.de" rel="noopener">www.john-doe.de</a>
<a href="http://john-doe.de" rel="noopener">john-doe.de</a>
--

Please mail me at <a href="mailto:demo@example.com">demo@example.com</a> or visit mit on:
- lorem ipsum <a href="http://github.com/nikolalohinski/gonja" rel="noopener">github.com/nikolalohinski/gonja</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de" rel="noopener">http://www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="https://www.john-doe.de" rel="noopener">https://www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="https://www.john-doe.de" rel="noopener">https://www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de" rel="noopener">www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de/test=%22test%22" rel="noopener">www.john-doe.de/test=&quot;test&quot;</a> lorem ipsum

--

Please mail me at <a href="mailto:demo@example.com">demo@example.com</a> or visit mit on:
- lorem ipsum <a href="http://github.com/nikolalohinski/gonja" rel="nofollow noopener" target="_blank">github.com/nikolalohinski/gonja</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de" rel="nofollow noopener" target="_blank">http://www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="https://www.john-doe.de" rel="nofollow noopener" target="_blank">https://www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="https://www.john-doe.de" rel="nofollow noopener" target="_blank">https://www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de" rel="nofollow noopener" target="_blank">www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de/test=%22test%22" rel="nofollow noopener" target="_blank">www.john-doe.de/test=&quot;test&quot;</a> lorem ipsum

--

Please mail me at <a href="mailto:demo@example.com">demo@example...</a> or visit mit on:
- lorem ipsum <a href="http://github.com/nikolalohinski/gonja" rel="noopener">github.com/n...</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de" rel="noopener">http://www.j...</a> lorem ipsum
- lorem ipsum <a href="https://www.john-doe.de" rel="noopener">https://www....</a> lorem ipsum
- lorem ipsum <a href="https://www.john-doe.de" rel="noopener">https://www....</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de" rel="noopener">www.john-doe.de</a> lorem ipsum
- lorem ipsum <a href="http://www.john-doe.de/test=%22test%22" rel="noopener">www.john-doe...</a> lorem ipsum

